dbt/models/shared_columns.md

```diff
@@ -649,7 +650,8 @@ Possible values for this variable are:
 {% docs shared_column_char_hbath %}
 Number of half baths.
 
-Defined as bathrooms without a shower or bathtub
+Defined as bathrooms without a shower or bathtub. Note that either 0 or
```

Comment:
**suggestion:** Let's make this same update for `char_porch`

Reply:
`char_porch` has a bulleted list of values, so I just added it there:

https://github.com/ccao-data/data-architecture/blob/250050ec5bc2b88bff74e98a993a85c864554c78/dbt/models/shared_columns.md?plain=1#L682-L687

Does that seem good enough to you, or should we add a prose description instead?